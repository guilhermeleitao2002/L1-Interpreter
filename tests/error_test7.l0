let x = 5;
if (x) {
  print x
} else {
  print 0
};;