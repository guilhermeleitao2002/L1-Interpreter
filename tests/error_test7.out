Runtime Error:
InterpreterError: Type error: if condition must be boolean